.github/workflows/test.yml

```diff
@@ -0,0 +1,65 @@
+name: ci
+
+on:
+  - push
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install package
+        run: pip install .[dev]
```

Comment:
I think I've occasionally slipped on this too; it turns out if the job _only_ runs `pre-commit`, you can probably avoid pip installing your altogether because pre-commit manages installing its hooks and their dependencies.